_maps/shuttles/tgs_canterbury.dmm

```diff
@@ -1028,17 +1043,19 @@
 	},
 /area/shuttle/canterbury/medical)
 "ch" = (
-/obj/machinery/light/small{
-	icon_state = "bulb1";
-	dir = 8
+/obj/machinery/atmospherics/pipe/simple/scrubbers/hidden{
+	icon_state = "pipe11-2";
```

Comment:
Looking at it a bit better, the map may already be full of var-edits.
If you can reduce them, that'd be nice.
If not, oh well.